routers/web/repo/pull.go

```diff
@@ -299,51 +299,51 @@ func ForkPost(ctx *context.Context) {
 	ctx.Redirect(ctxUser.HomeLink() + "/" + url.PathEscape(repo.Name))
 }
 
-func checkPullInfo(ctx *context.Context) *issues_model.Issue {
+func getPullInfo(ctx *context.Context) (*issues_model.Issue, bool) {
```

Comment:
What about named return values like
```suggestion
func getPullInfo(ctx *context.Context) (issue *issues_model.Issue, ok bool) {
```
?